### PR TITLE
Updated for Aternos changes + new Commands

### PR DESCRIPTION
**Pull request by brandonnly**
# Changes

Found that the bot just didn't work for me so I overhauled it a little. Aternos also changed its layout and removed the public page for servers.

## ADDED

- Bot status updates based on server status and player count
- `--crash` to easily close the bot
- Pings the user when their server has launched
- Variables for `headless` and `Adblock`
  - `Headless`: to easily turn off for testing purposes (`True` by default)
  - `Adblock`: I have a network-wide ad-blocker so I can't whitelist Aternos (`False` by default)
    - Simulates going through the 3-second wait
  - Adding these into the launch config window would be a good addition later
- `--server info` to return server info including the IP and server version
image

## CHANGED

- Aternos UI was updated and changed some of the xpaths--code was changed to reflect this
- Chromedriver-py requirements to version 87
  - Updated agent header to Chrome 87 as well
- General formatting changes to PEP8 and changed `.format` to `f""`
- Configuration window was resized so text input can fit nicely
- Chrome tab opens once and stays open while the bot is running
- `connect_and_launch.py` no longer checks for .env as `Bot.py` already does
- `README.me` was updated to reflect all new commands
  - Also added a note about cloud hosting as I spent hours failing to do it lol

## REMOVED

- Helper Functions (helper.py)
  - discord.py is designed so that coroutines decorated not using the bot/client decorators can only be run once ([see here](https://github.com/Rapptz/discord.py/issues/2079))
  - This meant that you had restart the bot each time you want to start/stop the server
- Server Status URL (removed from tkinter and files)
  - Aternos doesn't have the public-facing website for servers anymore

### CODE REVIEW

File: connect_and_launch.py [hunk 1]
```diff
@@ -1,96 +1,121 @@
...
-options.add_argument("user-agent=Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/81.0.4044.138 Safari/537.36 OPR/68.0.3618.125")
+if headless:
+    options.add_argument('--headless')
+options.add_argument("user-agent=Mozilla/5.0 (Windows NT 10.0; Win64; x64) "
+                     "AppleWebKit/537.36 (KHTML, like Gecko) "
+                     "Chrome/87.0.4280.88 Safari/537.36")
 
 driver = webdriver.Chrome(options=options, executable_path=binary_path)
 
-@can_fire_async
```
Comment (Mekolaos) [inline connect_and_launch.py:29]: I'm guessing this created quite a load of issues. Good riddance.

File: README.md [hunk 2]
```diff
@@ -4,21 +4,33 @@ A simple tool to serve your own discord bot so you can manage an Aternos server
...
-1. Install it using either:
+1. Git clone this repository
+2. Install it using either:
    * Copy, paste and execute this command inside the project folder: ```pip install -r requirements.txt```
-   * Alternatively you can create a virtual environement using ```python -m venv venv ``` and then ```source venv/bin/activate``` to finally ```pip install -r  requirements.txt```
-1. Execute using this command inside the project folder : python3 Bot.py
+   * Alternatively you can create a virtual environnement using ```python -m venv venv``` and then ```source venv/bin/activate``` to finally ```pip install -r  requirements.txt```
+3. Execute using this command inside the project folder : ```python3 Bot.py```
+
+**Note:** When running ```Bot.py``` for the first time, you'll be prompted to enter your Aternos account and bot token. You'll need to run ```python3 Bot.py``` again after you finish setting your information.
```
Comment (Mekolaos) [inline README.md:13]: Just wanna say thanks, maintaining documentation sucks, and I should've updated it a while ago.

**Review by Mekolaos — APPROVED** [review]
Best update possible, thanks for making this garbage repo better mate.

**Comment by brandonnly** [comment]
> Best update possible, thanks for making this garbage repo better mate.

No problem! My first open-source contribution and I learned selenium so I had a good time :)

One oversight on my part though:

`--stop` doesn't account for when the text says `'Offline'` -- only `Stopping ...` and `Saving ...`

**Comment by starmaid** [comment]
I had been looking at trying to fix the broken public page issues for a couple weeks but never got around to it. Thanks @brandonnly ! My server is going to be happy to have this up and running again :)

**Pull request merged**